### PR TITLE
NAS-109677 / 21.04 / Update repo info

### PR DESCRIPTION
**Pull request by denysbutenko**

### CODE REVIEW

File: README-egret.md [hunk 1]
```diff
@@ -1,64 +1,64 @@
...
-## Further help
-
-To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
+# TrueNAS WebUI
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.0.4. 
+
+## Angular / Material Design
+
+The FreeNAS WebUI is built with official Angular Material Design components and features:
```
Comment (dszidi) [inline README-egret.md:7]: Missed a spot

Comment (denysbutenko) [reply README-egret.md]: updated

File: package.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/freenas/webui.git"
+    "url": "https://github.com/truenas/webui.git"
   },
   "bugs": {
     "url": "https://bugs.freenas.org"
```
Comment (dszidi) [inline package.json:38]: Waiting to hear back from management about this link. There might be something more appropriate for this entry.

Comment (denysbutenko) [reply package.json]: Done 👍🏻

**Review by dszidi — APPROVED** [review]
LGTM

**Pull request merged**